test/cases/chunks/named-chunks/index.js

```diff
@@ -33,3 +33,46 @@ it("should handle empty named chunks", function(done) {
 		sync = false;
 	});
 });
+
+it("should be able to use named chunks in import()", function(done) {
+	var sync = false;
+  import("./empty?e" /* webpackChunkName = "import-named-chunk" */).then(function(result){
```

Comment:
My bad. Looks like didn't load the .editorconfig at the time. It's weird. I've reinstall my editor plugin then repaired it.